index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const util = require('util');
-const Writable = require('readable-stream/writable');
+const Writable = require('readable-stream/lib/_stream_writable.js');
```

Comment:
In [`readable-stream`'s usage note](https://github.com/nodejs/readable-stream/blob/v3.6.0/README.md#usage), they recommend something like `const { Writable } = require('readable-stream');` for compatibility.